src/models/ITestResult.ts

```diff
@@ -1,3 +1,5 @@
+import { DefectGETIVA } from '@dvsa/cvs-type-definitions/types/iva/defects/get';
```

Comment:
Not a comment specific to this piece of code, but seeing the model sparked a question in my mind. Has this been tested in the deployed code - has an IVA30 been submitted via Postman with the populated ivaDefects array and subsequently been accepted by the schema and recorded in the test-results table in AWS?